src/components/docs/Excerpt/index.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+import styles from './styles.module.css';
+
+const Excerpt = ({image, text}) => {
```

Comment:
Can we pass in the `text` as a child? I want to avoid dangerouslySetInnerHTML. Just wrap the child in a `p` tag?